packages/proxy/lib/http/request-middleware.ts

```diff
@@ -81,7 +81,7 @@ const MaybeEndRequestWithBufferedResponse: RequestMiddleware = function () {
 
   if (buffer) {
     this.debug('ending request with buffered response')
-    this.res.wantsInjection = 'full'
+    this.res.wantsInjection = buffer.isMultiDomain ? 'fullMultiDomain' : 'full'
```

Comment:
I'm completely unfamiliar with buffered responses. In what cases does this happen?

Reply:
When calling `cy.visit()`, a call is made to the backend to to resolve the url, essentially applying any passed in options and making the actual request but not returning the response. When the real request is then made from the browser (by either changing the iframe src or by changing top), the response is available here without having to make the request again.

Reply:
Ah OK that makes sense. Thank you!